autogpt_platform/frontend/src/components/agptui/Status.stories.tsx

```diff
@@ -44,6 +44,9 @@ export const Rejected: Story = {
 };
 
 export const AllStatuses: Story = {
+  args: {
```

Comment:
this is a funny requirement because how the story is defined. We could do this better but its fine for now